### ISSUE TITLE
[Feature Request] Are there some way to customize summary row or display html

### ISSUE THREAD

**Issue by Chellhooo** [ElemeFE/element#18438]
### Existing Component
是

### Component Name
Table

### Description
hi~ I want to show total with **fixed** row at the bottom of table, and it seems the only way is summary row.
But I also want to **customize** content with html, for example, i want to display two-line content in the cell.

Is possible that support summary slot or richtext in summary row?

你好，我想问一下，element现在有方法可以实现table组件最后一行固定的同时，支持自定义cell的内容吗（比如支持多行数据）? 

如果可以，希望可以考虑一下支持summary/footer slot，或者可以在summary/footer行支持富文本 ~

**Comment by eele94** [comment]
Found any solution for this?

**Comment by motishaku** [comment]
@eele94 Did you manage to figure this out?

**Comment by eele94** [comment]
> @eele94 Did you manage to figure this out?

Nope, in the end only used the show-summaries and summary-method(getSummaries) method as described in the documentation

**Comment by mubasshir** [comment]
@motishaku did you find any solution or way around ?
